src/components/navigation/navigation-item/NavigationItem.tsx

```diff
@@ -73,21 +73,14 @@ export class NavigationItem extends Component<NavigationItemProps, NavigationIte
   componentDidUpdate(previousProps: Readonly<NavigationItemProps>, nextState: Readonly<NavigationItemState>) {
     this._getSelected();
     this.matchHeight();
-    if (nextState.imageLoaded !== this.state.imageLoaded) {
```

Comment:
don't we need image load handling anymore?

Reply:
I think that titleTrimmed was only used to determine whether to show the show more text ? This is now handled by expandable-text